packages/x-tree-view/src/internals/plugins/useTreeViewExpansion/useTreeViewExpansion.test.tsx

```diff
@@ -0,0 +1,244 @@
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import { describeTreeView } from 'test/utils/tree-view/describeTreeView';
+import { UseTreeViewExpansionSignature } from '@mui/x-tree-view/internals';
+import { act, fireEvent } from '@mui-internal/test-utils';
+
+/**
+ * All tests related to keyboard navigation (e.g.: expanding using "Enter" and "ArrowRight")
+ * are located in the `useTreeViewKeyboardNavigation.test.tsx` file.
+ */
+describeTreeView<UseTreeViewExpansionSignature>('useTreeViewExpansion plugin', ({ render }) => {
+  describe('model props (expandedItems, defaultExpandedItems, onExpandedItemsChange)', () => {
```

Comment:
This `describe` should be replicated almost exactly for every other model.
We could almost create an abstraction for this at some point to make sure we always test models correctly (we did not do it on the grid and I would not be surprised if not all model were tested as strongly as the oldest ones).

But for now I think we can just all remember that a new model = a new describe that contains all the equivalent tests to the expansion ones.

If you see some missing tests, it can be interesting to add them before we merge and apply the same pattern to all the other models.